.github/release-drafter.yml

```diff
@@ -0,0 +1,52 @@
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: '# Breaking Changes'
+    labels:
+        - 'breaking change'
+  - title: '# Enhancements'
+    labels:
+      - 'type: enhancement'
+  - title: '# Performance'
+    labels:
+      - 'type: performance'
+  - title: '# Bugs Fixed'
+    labels:
+      - 'type: bug'
+  - title: '# Types'
+    labels:
+      - 'type: types'
+  - title: '# Documentation'
+    labels:
+      - 'type: documentation'
+  - title: '# Development'
+    labels:
+      - 'type: chore'
+change-template: '- #$NUMBER $TITLE'
+change-title-escapes: '\<*_&`#@'
+version-resolver:
+  major:
```

Comment:
What does this part do?

Reply:
It tries to figure out the next version number from the commit labels. Could be usable when in normal release cycle (out of beta)